### PR TITLE
Feature: add the thing

### PR DESCRIPTION
**Pull request by alexlande**
This adds an extremely simple CLI tool to generate valid react inline styles from a CSS source file. Some known limitations at this point (I'll open issues with these, but didn't want to block on making something available.):
- Doesn't handle media queries (just throws them out)
- Only works with a single input file and a single output file (no streams or arrays of sources or anything like that at this point)

/cc @colinmegill @ryan-roemer

### CODE REVIEW

**Comment by alexlande** [comment]
Thinking on this more, I'm going to push a couple of extra commits here to handle two cases better:
1. Media queries (fine I'll just do it ugh)
2. Multiple declarations with the same selector (ideally your CSS shouldn't include this, but it's possible! May as well account for it).

**Comment by alexlande** [comment]
Pushed changes so that this will handle media queries now. I also explored handling duplicate selectors, but I couldn't think of a valid use case for that, so instead I'm just merging any duplicate selectors together now, which behaves similarly to the way the browser would handle those styles anyway (with duplicate styles declared later taking precedence).

File: README.md [hunk 1]
```diff
@@ -43,3 +43,25 @@ module.exports = {
...
+## Options
+
+### quote
+
+`--quote=[type]`
+
+Type of quote used in generated JS. Defaults to `single`.
+
+```
+css-to-radium style.css style.js --quote=double
```
Comment (ryan-roemer) [inline README.md]: @alexlande -- Oh, and there isn't a `TODO` here, but this definitely needs an update ;)

**Comment by alexlande** [comment]
@ryan-roemer: Can I trouble you for one last review?

File: bin/cli.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+// ----------------------------------------------------------------------------
+var getTabString = function (tabWidth) {
+  var result = '';
+
+  for (var i = 0; i < tabWidth; i++) {
+    result += ' ';
+  }
+
+  return result;
+};
```
Comment (ryan-roemer) [inline bin/cli.js]: Remove this and below, just do:

``` js
indent: cliArgs.indentSize ? (new Array(cliArgs.indentSize)).join(' ') : '\t'
```

File: bin/cli.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+var stringifyResult = function (data) {
+  return stringifyObject(data, {
+    singleQuotes: cliArgs.quote === 'double' ? false : true,
```
Comment (ryan-roemer) [inline bin/cli.js]: Simplify to:

``` js
singleQuotes: cliArgs.quote !== 'double'
```

File: bin/cli.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  var converted = new Buffer(stringifyResult(cssObj));
+  this.push(converted);
+  callback();
+});
+
+// Encoding.
+process.stdin.setEncoding('utf8');
+
+// Input: Files or stdin.
+var inputStream = inputs ?
```
Comment (ryan-roemer) [inline bin/cli.js]: This is a little hard to read. I'd suggest:

``` js
var inputStream = process.stdin;
if (inputs) {
  inputStream = new Kat();
  inputs.forEach(function (input) {
    inputStream.add(input);
  });
}
```

**Comment by ryan-roemer** [comment]
@alexlande -- Some comments and suggestions, rest LGTM without re-review. Nice work!

**Pull request merged**